python/mujinplanningclient/planningclient.py

```diff
@@ -430,6 +430,46 @@ def SetCameraTransform(self, pose=None, transform=None, distanceToFocus=0.0, tim
         viewercommand.update(kwargs)
         return self.Configure({'viewercommand': viewercommand}, timeout=timeout, fireandforget=fireandforget, slaverequestid=slaverequestid)
 
+    #
+    # Crop-margin related
+    # 
+
+    def SetCropMarginVisible(self, name=None, visible=False, timeout=10, fireandforget=True, slaverequestid=None):
+        """
+        Sets the crop margin visualzation to be on or off
+
+        Args:
+            name: name of the kinbody
+            visible: boolean value setting the visibility of the crop margin visualization
+        """
```

Comment:
```suggestion
        """Sets the crop margin visualzation to be on or off.

        Args:
            name (str, optional): Name of the KinBody.
            visible (bool): If False, the crop margin is not visualized.
        """
```
Ibid for the other one.

Reply:
Improved docstring. Also combined the two apis into one.